src/generator/models.ts

```diff
@@ -5,183 +5,129 @@
 
 import { Session } from '@azure-tools/autorest-extension-base';
 import { comment, joinComma } from '@azure-tools/codegen'
-import { CodeModel, ObjectSchema, ChoiceSchema, Schema, SchemaType, StringSchema, Property } from '@azure-tools/codemodel';
+import { CodeModel, ObjectSchema, ChoiceSchema, Language, ChoiceValue, Schema, SchemaType, StringSchema, Property } from '@azure-tools/codemodel';
 import { values } from '@azure-tools/linq';
+import { ContentPreamble, ImportManager, SortAscending } from './helpers'
 
 // Creates the content in models.go
 export async function generateModels(session: Session<CodeModel>): Promise<string> {
-  const headerText = comment(await session.getValue("header-text", "MISSING LICENSE HEADER"), "// ");
-  const namespace = await session.getValue('namespace');
-  let text = `${headerText}\n\n`
-  text += `package ${namespace}\n\n`
+  let text = await ContentPreamble(session);
```

Comment:
When should we use let vs const

Reply:
Default should be to use `const`.  Only time you want to use `let` is if you plan to reassign some value to the variable.